Kirill Litvinov/script.js

```diff
@@ -0,0 +1,210 @@
+function Menu(price, calories) {
+    this.price = price;
+    this.calories = calories;
+}
+
+/**
+* Класс, объекты которого описывают параметры гамбургера. 
+* 
+* @constructor
+* @param size        Размер
+* @param stuffing    Начинка
+*/
+
+function Hamburger(size, stuffing) { 
+    Menu.call(this);
+    this.size = new Menu(size.price, size.calories);
+    this.stuffing = new Menu(stuffing.price, stuffing.calories);
```

Comment:
You create one instance of Hamburger and in the same time you create two instances of Menu. The main idea of prototyping is to reduce number of instances. You may separate methods of calculating calories and price, for Menu and for Hamburger, to prevent duplicating.